src/main/java/com/salesforce/dataloader/action/visitor/BulkApiVisitorUtil.java

```diff
@@ -169,10 +209,11 @@ boolean hasJob() {
         return this.jobInfo != null;
     }
 
-    void closeJob() throws AsyncApiException {
-        this.jobInfo = this.client.closeJob(getJobId());
+    void awaitCompletionAndCloseJob() throws AsyncApiException {
+        this.jobInfo = this.client.getJobStatus(getJobId());
```

Comment:
Does `getJobStatus` still return a `jobInfo`?

Reply:
Yes, it does. the only thing changed is name of the method. Semantics are same as before.